src/transforms/utils/ElmWrappers.ts

```diff
@@ -0,0 +1,22 @@
+const invocationRegex = /^A(?<arity>[1-9]+[0-9]*)$/;
+const wrapperRegex = /^F(?<arity>[1-9]+[0-9]*)$/;
```

Comment:
I'm almost wondering whether we should be more precise, and only match A2->A9 and F2-F9. Thoughts?

Reply:
That makes sense to me!  I can make that change